plotter/field_test.go

```diff
@@ -0,0 +1,146 @@
+// Copyright ©2019 The Gonum Authors. All rights reserved.
```

Comment:
To make the examples more legible in godoc, it might make sense to split the examples and tests up into individual files. From the [documentation](https://blog.golang.org/examples):

> A whole file example is a file that ends in _test.go and contains exactly one example function, no test or benchmark functions, and at least one other package-level declaration. When displaying such examples godoc will show the entire file.

Reply:
I think this is another PR. The example code that this is adapted from, heat_test.go is the same. In fact I think they all are because the `TestX` funcs are in the same files.